### PR TITLE
_

### LINKED ISSUE #42: jeremiahmegel - Pella, IA
## Pella, Iowa

- [ ] Create a branch
- [ ] Add file
- [ ] Create a commit
- [ ] Open a pull request
- [ ] Have a discussion about pull request
- [ ] Merge pull request

### PR DESCRIPTION
**Pull request by smmr0**

### CODE REVIEW

**Comment by jeremiahmegel** [comment]
@beardofedu 'Sup?

File: places/jeremiahmegel-pella.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+## Pella, IA
+
```
Comment (readiu) [inline places/jeremiahmegel-pella.md]: Who are you???

File: places/jeremiahmegel-pella.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+### Fun facts
```
Comment (jeremiahmegel) [inline places/jeremiahmegel-pella.md]: Fix this: there's nothing fun about Pella.

**Comment by jeremiahmegel** [comment]
I'm-a give myself the go-ahead: :shipit:

**Pull request merged**